src/tmpfiles/tmpfiles.c

```diff
@@ -3861,6 +3867,14 @@ static int parse_line(
                         if (!i.argument)
                                 return log_oom();
                 }
+
+                if (i.ignore_if_missing && access_nofollow(i.argument, F_OK) == -ENOENT) {
+                        /* Silently skip over lines where the source file is missing. */
+                        log_syntax(NULL, LOG_DEBUG, fname, line, 0,
+                                   "Symlink source path '%s' does not exist, skipping line.", i.argument);
+                        return 0;
```

Comment:
we should log about unexpected errors of chase_and_accessat() btw, at least at debug level.